tests/resources/VCH-Util.robot

```diff
@@ -412,6 +415,8 @@ Install VIC with version to Test Server
     ${rc}  ${output}=  Run And Return Rc And Output  tar zxvf vic.tar.gz
     Set Environment Variable  TEST_TIMEOUT  20m0s
     Install VIC Appliance To Test Server  vic-machine=./vic/vic-machine-linux  appliance-iso=./vic/appliance.iso  bootstrap-iso=./vic/bootstrap.iso  certs=${false}  vol=default ${insecureregistry}
+
+    Clean VMOMI Sessions  vic-machine  # old versions of vic-machine may leak sessions, confusing other tests
```

Comment:
I would think we might actually need this in the install keyword instead/in-addition of delete keyword, if we want to make sure to cleanup previous builds.